elastic/kibana
# [Guided onboarding] Dynamic URLs for steps

## Summary
Fixes https://github.com/elastic/kibana/issues/143322

This PR adds an option to store dynamic parameters when a step is completed to be used later for dynamically built URLs.

### How to test
1. Add `xpack.cloud.id: 'testID'` to your `/config/kibana.dev.yml` file
2. Start ES with `yarn es snapshot`
3. Start Kibana with `yarn start --run-examples` 
4. Navigate to the guided onboarding example plugin `http://localhost:5601/app/guidedOnboardingExample`
5. Start the test guide and when completing step 1 provide any value for the paramter `indexName`
6. Continue the guide until step 4 and check that the correct value is being used for the url of this step. 

### Checklist
